src/renderer/extensions/vueNodes/widgets/components/WidgetGalleria.vue

```diff
@@ -25,16 +24,16 @@
     >
       <template #item="{ item }">
         <img
-          :src="item.itemImageSrc || item.src || item"
-          :alt="item.alt || 'Gallery image'"
+          :src="item?.itemImageSrc || item?.src || ''"
+          :alt="item?.alt || t('Gallery image')"
```

Comment:
This might not sound very great for screen readers.
Tabbing through would be pretty repetitive and you wouldn't necessarily know how far you'd gone or how far you had to go.